scripts/demo_content.py

```diff
@@ -0,0 +1,407 @@
+"""
+Drop any test database tables (user, item, order, orderitem)
+and supply a new one db with new down-to-earth data.
+"""
+
+from peewee import SqliteDatabase
+
+from models import User, Item, Order, OrderItem
+
+
+db = SqliteDatabase('database.db', autocommit=False)
+
+
+def main():
+
+    def drops_all_tables(database):
+        if db.database == 'database.db':
+            tables = db.get_tables()
+            for table in tables:
+                if table == 'item':
+                    Item.drop_table()
+                if table == 'order':
+                    Order.drop_table()
+                if table == 'user':
+                    User.drop_table()
+                if table == 'orderitem':
+                    OrderItem.drop_table()
```

Comment:
Ho provato a fare questo, ma c'è un comportamento derivato dalla riga 2 'from models' che non mi aspettavo. Questa riga mi crea automaticamente un file db vuoto de nome database.db se il file database.db non esiste. Quindi mi viene da pensare che creare db da nomi diversi sia un po come girare in vuoto.  Comunque, continuerò a provare una soluzione a questo punto del task.